db.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+    "log"
+    "labix.org/v2/mgo"
+    "time"
+)
+
+// Save the session here for easy access
+var mongoSession *mgo.Session
+
+// Details we need to connect to the database
+// @TODO: Let's use variables from ENV, like
+// os.Getenv("DB_HOST")
+const (
+    // Host of database to connect to
```

Comment:
Maybe remove this comment, and the four below? As they are now the comments convey basically the same thing as the variable names, and they are thus IMO somewhat uneeded.